src/config.rs

```diff
@@ -232,11 +233,23 @@ impl Config {
             .cookie_file
             .unwrap_or_else(|| daemon_dir.join(".cookie"));
 
+        let daemon_auth = match config.auth {
```

Comment:
There's no check whether both option are specified - that should be rejected as ambiguous. I usually do that by writing `match (config.cookie_file, confi.auth)` and enumerating all possibilities.

Reply:
Sounds good, fixed.